model-optimizer/mo/graph/perm_inputs.py

```diff
@@ -171,6 +171,24 @@ def transpose(op_node: Node, port_info: str, input_port: int):
     op_node.in_port(input_port).get_connection().insert_node(transpose)
 
 
+def transpose_nchw_to_nhwc(op_node: Node, port_info: str, input_port: int):
```

Comment:
It does not work properly for my operation `GatherND`, namely, shape prior Transpose is incorrectly computed, if I insert this rule in `shape_infer` function.
May `shape_inference=True` is missed for it?